netctl/http.go

```diff
@@ -54,7 +54,10 @@ func postMap(ctx *cli.Context, url string, jsonMap map[string]interface{}) {
 
 	resp, err := client.Post(url, "application/json", bytes.NewBuffer(content))
 	if err != nil {
-		writeBody(resp, ctx)
+		if resp != nil {
```

Comment:
Well, I thought it would be a useful debug tool. Perhaps we can just have `--debug` cli flag that has this behavior